packages/atomic/.storybook/main.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
   webpack: (config) => {
     return {
       ...config,
+      performance: {
```

Comment:
So yeah, the storybook website is not particularly optimized :D

Reply:
It displays errors as well? Damn. Otherwise, setting hints to `'error'` would be nice